ui/org.eclipse.pde.core/src/org/eclipse/pde/internal/core/builders/ManifestConsistencyChecker.java

```diff
@@ -372,13 +375,21 @@ private void validateBuildPropertiesExists(IProject project) {
 			}
 			// if build.properties doesn't exist and build problems != IGNORE, create a marker on the project bug 172451
 			try {
+				if (!project.hasNature(PDE.PLUGIN_NATURE)) {
+					// hod did this happen?
```

Comment:
I assume this is temporary so the hod versus how is not so important.

Reply:
thanks for mentioning this typos